### DISCUSSION TITLE
Is there a way to change the UI to a "light" skin rather than the default "black ops"?

### DISCUSSION THREAD

**Discussion by CalculonPrime** [oobabooga/text-generation-webui#4060] [Q&A] [answered]
I prefer the "light" skin like Automatic1111's Web UI, not the "black ops" dark style.  Can I change this?

**Comment by missionfloyd** [comment, marked as answer]
Click the "Toggle 💡" button on the Session tab.

**Comment by CalculonPrime** [comment]
I did look on the "sessions" tab but didn't see anything promising.  Maybe it should say "Toggle Light/Dark Skin" or "Toggle Light/Dark Theme", or perhaps it should have a picklist there for the theme.

**Reply by lursel** [reply]
Just as an FYI for future readers, it does say "Toggle light/dark theme💡" now.
